### PR TITLE
[DEMO] MOCO-12: Change InvoiceID into InvoiceItemID

### PR DESCRIPTION
**Pull request by robdev91**

### CODE REVIEW

File: app/src/lib/queries/graphqlGenerated.tsx [hunk 1]
```diff
@@ -1747,7 +1747,7 @@ export type QueryOrganizationArgs = {
 
 
 export type QueryOrganizationByIdArgs = {
-  id?: InputMaybe<Scalars['UUID1']>;
+  id: Scalars['UUID1'];
```
Comment (Dmckinney821) [inline app/src/lib/queries/graphqlGenerated.tsx:1750]: Why remove the `?`

File: package.json [hunk 2]
```diff
@@ -86,7 +86,7 @@
     "@graphql-codegen/typescript": "^2.4.11",
     "@graphql-codegen/typescript-operations": "^2.4.0",
     "@graphql-codegen/typescript-react-apollo": "^3.2.14",
-    "@rollup/plugin-commonjs": "^22.0.0",
+    "@rollup/plugin-commonjs": "^21.1.0",
```
Comment (Dmckinney821) [inline package.json:89]: you downgraded

Comment (robdev91) [reply package.json:89]: yes

Comment (robdev91) [reply package.json:89]: rollup should be downgraded for build issue

File: app/src/lib/queries/graphqlGenerated.tsx (hunk 1 above)
Comment (robdev91) [reply app/src/lib/queries/graphqlGenerated.tsx:1750]: seems like changed in BE. just run `yarn codegen`

**Review by Danziger — APPROVED** [review]

File: rollup.config.js [hunk 3]
```diff
@@ -1,53 +1,20 @@
 import commonjs from "@rollup/plugin-commonjs";
+import dts from "rollup-plugin-dts"
 import resolve from "@rollup/plugin-node-resolve";
-import * as fs from 'fs';
```
Comment (Danziger) [inline rollup.config.js:3]: @robdev91 Can you please restore these changes?

**Pull request merged**